### PR TITLE
refac(node): do refactors in code structure of chain_spec

### PR DESCRIPTION
**Pull request by aripiprazole**

### CODE REVIEW

File: node/src/chain_spec.rs [hunk 1]
```diff
@@ -253,38 +208,30 @@ fn network_genesis(
             balances: balances.to_vec(),
         },
         aura: AuraConfig {
-            authorities: initial_authorities.iter().map(|x| (x.0.clone())).collect(),
+            authorities: initial_authorities.iter().map(|(x, _)| (x.clone())).collect(),
```
Comment (saiintbrisson) [inline node/src/chain_spec.rs:211]:
```suggestion
            let (aura, grandpa): (Vec<_>, Vec<_>) = initial_authorities.iter().map(|(aura, grandpa)| (aura, (grandpa, 1))).unzaip(),
```

File: node/src/chain_spec.rs [hunk 2]
```diff
@@ -100,78 +100,37 @@ pub fn generate_config(network: String) -> Result<ChainSpec, String> {
...
-                module.2.as_bytes().to_vec(),                     // address
-                module.3.iter().map(|(a, b)| (*a, *b)).collect(), // weights: Convert to tuples
+                new_account_id(id),
+                name.as_bytes().to_vec(),
+                address.as_bytes().to_vec(),
+                weights.iter().map(|(a, b)| (*a, *b)).collect(),
             ));
         }
+
+        // Add stake to
```
Comment (saiintbrisson) [inline node/src/chain_spec.rs:119]:
```rust
let mut subnet_modules = Vec::with_capacity(state.modules[netuid].len());
for ... {
  subnet_modules.push((new_account_id, ...));
}
modules.push(subnet_modules);
```

File: node/src/chain_spec.rs [hunk 3]
```diff
@@ -100,78 +100,37 @@ pub fn generate_config(network: String) -> Result<ChainSpec, String> {
...
         stake_to.push(Vec::new());
-        for (key_str, key_stake_to) in state.stake_to[netuid].iter() {
+        for (key, key_stake_to) in state.stake_to[netuid].iter() {
```
Comment (saiintbrisson) [inline node/src/chain_spec.rs:121]: Same change here

File: node/src/chain_spec.rs (hunk 2 above)
Comment (aripiprazole) [reply node/src/chain_spec.rs:119]: Where am I supposed to write this?

**Pull request closed without merging**